kv/txn_coord_sender.go

```diff
@@ -296,15 +317,16 @@ func (tc *TxnCoordSender) sendOne(call client.Call) {
 		var txnMeta *txnMetadata
 		if txnMeta, ok = tc.txns[string(header.Txn.ID)]; !ok {
 			txnMeta = &txnMetadata{
-				txn:             *header.Txn,
+				txn:             header.Txn,
```

Comment:
This used to make a copy, and now it doesn't. Is there a danger that the caller may reuse the protos after this call and change the data out from under us?